pkg/controller/projectclaim/projectclaim_controller.go

```diff
@@ -101,55 +104,33 @@ func (r *ReconcileProjectClaim) Reconcile(request reconcile.Request) (reconcile.
 	return result, err
 }
 
+type ReconcileOperation func() (util.OperationResult, error)
```

Comment:
I don't think I understand that. Can you explain a bit what purpose it fits?
Should this be `util.OperationResult, error` or `gcputil.OperationResult, error`

Reply:
that's confusing. Here we import that package as util.